### PR TITLE
Add API homepage

### PR DESCRIPTION
**Pull request by SuperShadowPlay**
Addresses issue #21 

Added a folder (`src/homepage`) that contains a default landing page for the api server.

### CODE REVIEW

File: api-server/src/homepage/index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    to{opacity: .0;}
+  }
+
+  ::selection{
+    background: black;
+  }
+
+.death-button {
+  -webkit-border-radius: 25;
+  -moz-border-radius: 25;
```
Comment (jaythomas) [inline api-server/src/homepage/index.html:83]: Pretty sure you can get away with just `border-radius` on any browser made in the last 16 years without the need for these vendor prefixes. https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius#browser_compatibility

File: api-server/src/homepage/index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  transform: translate(-50%, -50%);
+  padding: 20px;
+  line-height: 30px;
+
+  font-size: 4em;
+  font-family: 'Courier New', Courier, monospace;
+  white-space: nowrap;
+  overflow: hidden;
+  width: 16em;
+  animation: type 4s steps(60, end);
```
Comment (jaythomas) [inline api-server/src/homepage/index.html:52]: Whitespace and indentation :snake: –  [Prettier](https://prettier.io/) plugin for your editor should auto-correct this.

File: api-server/src/homepage/index.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  font-size: 3em;
+  background: #b30000;
+  padding: 10px 20px 10px 20px;
+  border: solid #000000 2px;
+  text-decoration: none;
+}
+
+.death-button:hover {
+  background: #8c0303;
+  text-decoration: none;
```
Comment (jaythomas) [inline api-server/src/homepage/index.html:96]: Text decoration of `none` should be inherited from the base `.death-button` class, no?

File: api-server/src/homepage/index.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+<body>
+  <div class="overlord">
+    <div class="welcome-text">
+      Secret Database Dashboard<span class="blink"> &#9610;</span>
+    </div>
+
+    <div class="buttons-container">
+      <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">
+        <button type="button" class="death-button">Delete Database</button>
```
Comment (jaythomas) [inline api-server/src/homepage/index.html:15]: Consider using an `onclick()` here too to mask the URL. Adds to the suspense.

File: api-server/src/homepage/index.html [hunk 5]
```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"/>
+</head>
+
+<body>
+  <div class="overlord">
```
Comment (jaythomas) [inline api-server/src/homepage/index.html:8]: Unused class?

**Review by jaythomas — CHANGES_REQUESTED** [review]
Almost perfection...

File: api-server/src/homepage/index.html (hunk 5 above)
Comment (SuperShadowPlay) [reply api-server/src/homepage/index.html:8]: Yeah I think that was from some testing that I forgot about.

**Pull request merged**